plonetheme/onegov/viewlets/configure.zcml

```diff
@@ -11,23 +11,6 @@
       layer="plonetheme.onegov.interfaces.IPlonethemeOneGovLayer"
       />
 
-  <!-- The logo -->
-  <browser:viewlet
-      name="plone.logo"
-      manager="plone.app.layout.viewlets.interfaces.IPortalHeader"
-      class=".logo.LogoViewlet"
-      permission="zope2.View"
-      layer="plonetheme.onegov.interfaces.IPlonethemeOneGovLayer"
-      />
-
-  <browser:viewlet
-      name="plone.links.favicon"
-      manager="plone.app.layout.viewlets.interfaces.IHtmlHeadLinks"
-      class=".meta.MetaViewlet"
```

Comment:
Could we also remove the file `plonetheme/onegov/viewlets/meta.py`?

Reply:
And the file `plonetheme/onegov/viewlets/meta.pt` too?

Reply:
👍